spec/lib/pdf_fill/extras_generator_spec.rb

```diff
@@ -5,7 +5,7 @@
 
 describe PdfFill::ExtrasGenerator do
   subject do
-    described_class.new
+    described_class.new(start_page: 8)
```

Comment:
I think this diff isn't necessary, right? All the page number agnostic tests should still work if it defaults to 1.